media-gfx/img2pdf/files/img2pdf-0.4.0_endianness.patch

```diff
@@ -0,0 +1,245 @@
+@@ -3036,7 +3036,7 @@ def tiff_palette1_img(tmp_path_factory,
+     }, str(identify)
+     assert identify[0]["image"].get("colorspace") == "sRGB", str(identify)
+     assert identify[0]["image"].get("type") == "Palette", str(identify)
+-    assert identify[0]["image"].get("endianess") in ["Undefined", "LSB",], str(
++    assert identify[0]["image"].get("endianness") in ["Undefined", "LSB",], str(
+         identify
+     )  # FIXME: should be LSB
+     assert identify[0]["image"].get("depth") == 8, str(identify)
+@@ -3087,7 +3087,7 @@ def tiff_palette2_img(tmp_path_factory,
+     }, str(identify)
+     assert identify[0]["image"].get("colorspace") == "sRGB", str(identify)
+     assert identify[0]["image"].get("type") == "Palette", str(identify)
+-    assert identify[0]["image"].get("endianess") in ["Undefined", "LSB",], str(
++    assert identify[0]["image"].get("endianness") in ["Undefined", "LSB",], str(
+         identify
+     )  # FIXME: should be LSB
+     assert identify[0]["image"].get("depth") == 8, str(identify)
+@@ -3138,7 +3138,7 @@ def tiff_palette4_img(tmp_path_factory,
+     }, str(identify)
+     assert identify[0]["image"].get("colorspace") == "sRGB", str(identify)
+     assert identify[0]["image"].get("type") == "Palette", str(identify)
+-    assert identify[0]["image"].get("endianess") in ["Undefined", "LSB",], str(
++    assert identify[0]["image"].get("endianness") in ["Undefined", "LSB",], str(
+         identify
+     )  # FIXME: should be LSB
+     assert identify[0]["image"].get("depth") == 8, str(identify)
+@@ -3189,7 +3189,7 @@ def tiff_palette8_img(tmp_path_factory,
+     }, str(identify)
+     assert identify[0]["image"].get("colorspace") == "sRGB", str(identify)
+     assert identify[0]["image"].get("type") == "Palette", str(identify)
+-    assert identify[0]["image"].get("endianess") in ["Undefined", "LSB",], str(
++    assert identify[0]["image"].get("endianness") in ["Undefined", "LSB",], str(
+         identify
+     )  # FIXME: should be LSB
+     assert identify[0]["image"].get("depth") == 8, str(identify)
+@@ -3253,7 +3253,7 @@ def tiff_ccitt_lsb_m2l_white_img(tmp_pat
+     }, str(identify)
+     assert identify[0]["image"].get("colorspace") == "Gray", str(identify)
+     assert identify[0]["image"].get("type") == "Bilevel", str(identify)
+-    assert identify[0]["image"].get("endianess") in ["Undefined", "LSB",], str(
++    assert identify[0]["image"].get("endianness") in ["Undefined", "LSB",], str(
+         identify
+     )  # FIXME: should be LSB
+     assert identify[0]["image"].get("depth") == 1, str(identify)
+@@ -3334,7 +3334,7 @@ def tiff_ccitt_msb_m2l_white_img(tmp_pat
+     }, str(identify)
+     assert identify[0]["image"].get("colorspace") == "Gray", str(identify)
+     assert identify[0]["image"].get("type") == "Bilevel", str(identify)
+-    assert identify[0]["image"].get("endianess") in [
++    assert identify[0]["image"].get("endianness") in [
```

Comment:
Is this `endianess` just a typo across this file? I'd say `sed` works better for this than a patch file.